tsdb/wal/wal_test.go

```diff
@@ -475,3 +481,75 @@ func BenchmarkWAL_Log(b *testing.B) {
 		})
 	}
 }
+
+func TestWAL_Size(t *testing.T) {
```

Comment:
do we still need this test?

Reply:
You're right, this one does do an additional truncate, but that isn't really something that needs to be tested here. I'll remove it.